merqube_client_lib/secapi/client.py

```diff
@@ -19,9 +22,9 @@
 from merqube_client_lib.util import batch_post_payload
 
 
-class _ClientBase:
+class SecAPIClient(MerqubeApiClientBase):
```

Comment:
since we only have one class here, i eliminated the subclass, made no sense i suppose